watchdog/watchdog.go

```diff
@@ -72,7 +73,7 @@ func execute(ep *endpoint.Endpoint, alertingConfig *alerting.Config, maintenance
 	logr.Debugf("[watchdog.execute] Monitoring group=%s; endpoint=%s; key=%s", ep.Group, ep.Name, ep.Key())
 	result := ep.EvaluateHealth()
 	if enabledMetrics {
-		metrics.PublishMetricsForEndpoint(ep, result)
+		metrics.PublishMetricsForEndpoint(labels, ep, result)
```

Comment:
Same thing here, could you move the `labels` parameter to the end of the function's parameters list?

Reply:
85c5e732